pkg/machines/libvirt.es6

```diff
@@ -595,4 +593,25 @@ function buildConsoleVVFile(consoleDetail) {
         'fullscreen=0\n';
 }
 
+function doUsagePolling (name, connectionName) {
+    logDebug(`doUsagePolling(${name}, ${connectionName})`);
+
+    const canFailHandler = ({exception, data}) => {
+        console.info(`The 'virsh' command failed, as expected: "${JSON.stringify(exception)}", data: "${JSON.stringify(data)}"`);
+    };
+
+    return dispatch => {
```

Comment:
Indeed, I was missing that. ♩ It's a kind of maaa-giiiic ♪ ♫ ☺

Updated, it indeed looks much nicer now.